app/src/main/java/org/mozilla/focus/widget/AdjustBrightnessDialog.kt

```diff
@@ -27,6 +30,29 @@ class AdjustBrightnessDialog : BaseActivity() {
         }
     }
 
+    object Intents {
```

Comment:
Sorry, my bad. I didn't think through enough.
Base on the simple responsibility principle, AdjustBrightnessDialog shouldn't contain utility method and both dialog logic. We should be this object standalone in the package.
Maybe move the dialog and this object to a new package in "org.mozilla.rocket.nightmode"

Reply:
Updated.